website/docs/cli/commands/taint.mdx

```diff
@@ -16,7 +16,7 @@ propose to replace it in the next plan you create.
 
 ## Recommended Alternative
 
-For Terraform v0.15.2 and later, we recommend using the [`-replace` option](/cli/commands/plan#replace-address) with `terraform apply` to force Terraform to replace an object even though there are no configuration changes that would require it.
+For Terraform v0.15.2 and later, we recommend using the [`-replace` option](/cli/commands/plan#replace-address) with `terraform apply` to force Terraform to replace an object even though there are no configuration changes that would require it. 
```

Comment:
This is very much a nit, but as far as I can tell from GitHub's rendering here it seems like this was unchanged aside from adding an invisible space to the end of the line, which I'm guessing was an accident.

Reply:
hahah yes, it was an accident - let me fix that